waku/v2/protocol/waku_rln_relay/rln_types.nim

```diff
@@ -0,0 +1,23 @@
+import 
```

Comment:
This module is created to resolve a circular dependency issue.

Reply:
Can you put all types + logic associated (all `proc`s whose first argument is `RLN`) inside of an `rln.nim` module?

_The code that changes together should be together._